mac os error "use of undeclared identifier 'NSWindowStyleMaskClosable'"

when I build it on mac os ,error occurred 
```
error: 
      use of undeclared identifier 'NSWindowStyleMaskClosable'
  ...NSClosableWindowMask backing:NSBackingStoreBuffered defer:YES];
     ^
/Users/lgy/Documents/workspace/Pangolin/src/display/device/display_osx.mm:41:39: note: 
      expanded from macro 'NSClosableWindowMask'
#  define NSClosableWindowMask        NSWindowStyleMaskClosable
                                      ^
5 errors generated.
make[2]: *** [src/CMakeFiles/pangolin.dir/display/device/display_osx.mm.o] Error 1
make[1]: *** [src/CMakeFiles/pangolin.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/display/device/PangolinNSApplication.mm
+++ b/src/display/device/PangolinNSApplication.mm
@@ -29,7 +29,7 @@
 #include <pangolin/display/display.h>
 #include <pangolin/display/device/PangolinNSApplication.h>
 
-#if MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_12
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= 101200
 #  define NSAnyEventMask NSEventMaskAny
 #endif
 

--- a/src/display/device/PangolinNSGLView.mm
+++ b/src/display/device/PangolinNSGLView.mm
@@ -5,7 +5,7 @@
 #include <pangolin/display/display_internal.h>
 #include <pangolin/handler/handler_enums.h>
 
-#if MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_12
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= 101200
 #  define NSDeviceIndependentModifierFlagsMask NSEventModifierFlagDeviceIndependentFlagsMask
 #  define  NSShiftKeyMask NSEventModifierFlagShift
 #  define  NSControlKeyMask NSEventModifierFlagControl

--- a/src/display/device/display_osx.mm
+++ b/src/display/device/display_osx.mm
@@ -35,7 +35,7 @@
 #include <pangolin/display/device/PangolinNSApplication.h>
 #include <memory>
 
-#if MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_12
+#if MAC_OS_X_VERSION_MAX_ALLOWED >= 101200
 #  define NSFullScreenWindowMask      NSWindowStyleMaskFullScreen
 #  define NSTitledWindowMask          NSWindowStyleMaskTitled
 #  define NSMiniaturizableWindowMask  NSWindowStyleMaskMiniaturizable

